contract/src/main.rs

```diff
@@ -800,67 +793,34 @@ pub extern "C" fn set_approval_for_all() {
 
     let caller: Key = utils::get_verified_caller().unwrap_or_revert();
```

Comment:
maybe move to inside `get_all_owned_token_identifiers` method

Reply:
Moved inside the function